src/test/java/com/webank/webase/front/util/SampleKeys.java

```diff
@@ -0,0 +1,30 @@
+package com.webank.webase.front.util;
+
+import java.math.BigInteger;
+
+import org.fisco.bcos.web3j.crypto.Credentials;
+import org.fisco.bcos.web3j.crypto.ECKeyPair;
+import org.fisco.bcos.web3j.utils.Numeric;
+
+/** Keys generated for unit testing purposes. */
+public class SampleKeys {
+
+    public static final String PRIVATE_KEY_STRING =
+            "a392604efc2fad9c0b3da43b5f698a2e3f270f170d859912be0d54742275c5f6";
+    static final String PUBLIC_KEY_STRING =
+            "0x506bc1dc099358e5137292f4efdd57e400f29ba5132aa5d12b18dac1c1f6aab"
+                    + "a645c0b7b58158babbfa6c6cd5a48aa7340a8749176b120e8516216787a13dc76";
+    public static final String ADDRESS = "0xef678007d18427e6022059dbc264f27507cd1ffc";
+    public static final String ADDRESS_NO_PREFIX = Numeric.cleanHexPrefix(ADDRESS);
+
+    public static final String PASSWORD = "Insecure Pa55w0rd";
+
+    static final BigInteger PRIVATE_KEY = Numeric.toBigInt(PRIVATE_KEY_STRING);
+    static final BigInteger PUBLIC_KEY = Numeric.toBigInt(PUBLIC_KEY_STRING);
+
+    static final ECKeyPair KEY_PAIR = new ECKeyPair(PRIVATE_KEY, PUBLIC_KEY);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=4823531)